test/unit/frontend/github-configuration.test.ts

```diff
@@ -126,9 +131,14 @@ describe("Github Configuration", () => {
 		});
 	});
 
-	// TODO: try to figure out why this keeps giving a JSON web token decode error
-	describe.skip("#GET", () => {
+	describe("#GET", () => {
 		it("should return 200 when calling with valid Github Token", async () => {
+
+			// Don't use new github client for now - need better way to test with it
+			when(booleanFlag)
+				.calledWith(BooleanFlags.USE_NEW_GITHUB_CLIENT_TO_COUNT_REPOS, true, jiraHost)
```

Comment:
It looks like this flag was mistakenly added back to the feature-flags.js

This one was cleaned up last week

 - [ ] remove USE_NEW_GITHUB_CLIENT_TO_COUNT_REPOS flag

Reply:
cool, bad merge then.  I'll fix.